rfcs/trust-and-safety/20230109-trust_and_safety_preliminary_overview_and_exploration.md

```diff
@@ -0,0 +1,285 @@
+offensive or inaccurate way. Especially common examples of this are images of
+racialised people with original captions that use slurs or other derogatory
+language to describe the subject. I (Sara) do not know if this is something that
+currently exists in Openverse, but it is something we could discuss with
+providers if we discovered it to ensure that we are capturing the relevant
+metadata. For example, some providers may include a note clarifying that a
+caption or title of a work is "from the original source" rather than the
+museum's own description of a collection item. In these cases, it would be
+imperative for Openverse to also include that information if we surface the
+relevant text fields because it may serve double duty of giving historical
+context for a potentially sensitive and controversial cultural artefact _and_ as
+a manually attached content warning. Once Openverse has a way of scanning our
+catalogue for sensitive textual material, if we discover any of it to be coming
+from GLAM institutions, we should keep this in mind and work with the
+institution to understand how we can appropriately represent these sensitive
+historical materials.
+
+### The difference between offensive, sensitive, and illegal
+
+Openverse could choose to take different approaches to material that is
+considered to be variously offensive, sensitive, or illegal. For illegal
+materials, we should just remove it as much as possible and prevent its
+accidental reinclusion. This is clear and is not just a legal requirement but is
+also a baseline responsibility that we should consider. However, the definition
+of illegal probably depends somewhat (though not entirely, as there are certain
+classes of illegal materials that are essentially universally agreed upon) on
+what jurisdictions we need to abide by. Certain classifications of "illegal"
+material may not correspond to Openverse's priorities, for example, if a state
+entity attempts to use "illegality" to censor criticism. Luckily, Openverse does
+not need to make a distinct decision in this case as we can fall back to the
+WordPress Foundation's policies in this regard.
+
+When it comes to offensive and sensitive materials, however, the issue is more
+complicated. There are probably certain sensitive materials which are not
+illegal but that Openverse does not want to distribute. The WordPress Foundation
+probably already has a position on this kind of thing, and we should lean on
+pre-existing definitions and policies in that regard. For everything else,
+however, we'll need to make our own decisions about how to represent the
+material. One option is to generously blur visual content that may be offensive
+or otherwise sensitive. Additionally, adding content warnings as much as
+possible, both for visual and textual material.
+
+## Process user stories
+
+1. As an Openverse user, I expect results that are removed from upstream
```

Comment:
Any ideas on _how_ we can do that? When we re-ingest the items, we add the new items, update the existing ones, and do not remove the ones that were not present for the upstream provider. I don't think there's a way to know why the item was removed from the provider.
I wonder if adding some sort of scanning process for items that are not present at the provider when re-ingesting would help. We should also consider adding an "antitboosting" parameter (is there a word for the action that is opposite of search rank boosting?) to all of such items, assuming that they were removed from the provider for some reason.